RoadTripMoustache/kana_to_kanji
# Settings view

## 📖 Description


Add the settings view and a way to access it from the _Prepare Quiz_ view

### ⁉️ Related Issues




Closes #30 


### 🖼️ Screenshots:


<details>
    <summary>Full demonstration</summary> 
    
<img src="https://github.com/RoadTripMoustache/kana_to_kanji/assets/22211097/c375188c-0fdf-4eec-baae-ac07a09529da" width=300 />

</details>

### 🧪 How to test the change?




1. Open the application
2. On the prepare quiz view, click on the ⚙️ 
3. See the settings view

### ☑️ Checklist before requesting a review
